src/components/settings/Settings.h

```diff
@@ -69,13 +75,38 @@ namespace Pinetime {
       };
 
       void setWakeUpMode(WakeUpMode wakeUp) {
```

Comment:
Changing `setWakeUpMode(WakeUpMode wakeUp)` to `setWakeUpMode(WakeUpMode wakeUp, bool enable)` and setting the relevant bit of the `bitset` to `enable` would eliminate the need for a separate `unsetWakeUpMode` method.

Reply:
Good idea!